### PR TITLE
Hide new acronym if it's not approved

### PR DESCRIPTION
**Pull request by ne-cabrerav**

### CODE REVIEW

File: lambda/explain.py [hunk 1]
```diff
@@ -26,20 +27,16 @@
...
-        retval = item['Acronym'] + " - " + item['Definition'] + "\n---\n*Meaning*: " + item['Meaning'] +  "\n*Notes*: " + item['Notes']
-        
-    except:
-        retval = f'{acronym} is not defined.'
 
-    return retval
+        if item[APPROVAL_STR] == APPROVAL_STATUS_APPROVED:
+            return item['Acronym'] + " - " + item['Definition'] + "\n---\n*Meaning*: " + item['Meaning'] +  "\n*Notes*: " + item['Notes']
+        
+    return f'{acronym} is not defined.'
```
Comment (oijusti) [inline lambda/explain.py:39]: I think we are missing the case of acronyms that are added by default and have blank the approval attribute.

Comment (ne-cabrerav) [reply lambda/explain.py:40]: Done

**Review by oijusti — APPROVED** [review]

File: lambda/explain.py [hunk 2]
```diff
@@ -26,20 +27,17 @@
...
-        
-        retval = item['Acronym'] + " - " + item['Definition'] + "\n---\n*Meaning*: " + item['Meaning'] +  "\n*Notes*: " + item['Notes']
-        
-    except:
-        retval = f'{acronym} is not defined.'
 
-    return retval
+        approval = item.get(APPROVAL_STR)
+        if approval == None or approval == APPROVAL_STATUS_APPROVED:
+            return item['Acronym'] + " - " + item['Definition'] + "\n---\n*Meaning*: " + item['Meaning'] +  "\n*Notes*: " + item['Notes']
```
Comment (lapv1992) [inline lambda/explain.py:38]: can this be refactor using f-String?

**Review by lapv1992 — APPROVED** [review]

File: lambda/explain.py (hunk 2 above)
Comment (ne-cabrerav) [reply lambda/explain.py:38]: Yes, will do

**Pull request merged**